static/src/javascripts/projects/common/modules/navigation/sticky.js

```diff
@@ -68,25 +69,12 @@ define([
             }
         }
 
-        if (this.isMobile) {
-            mediator.on('window:scroll', _.throttle(function () {
-                this.updatePositionMobile();
-            }.bind(this), 10));
-        } else if (this.isAdblockInUse) {
-            mediator.on('window:scroll', _.throttle(function () {
-                this.updatePositionAdblock();
-            }.bind(this), 10));
-        } else if (this.isAppleCampaign) {
-            mediator.on('window:scroll', _.throttle(function () {
-                this.updatePositionApple();
-            }.bind(this), 10));
-        } else if (this.isProfilePage) {
-            this.updatePositionProfile();
-        } else {
-            mediator.on('window:scroll', _.throttle(function () {
-                this.updatePosition();
-            }.bind(this), 10));
-        }
+        // Get the name of the method to run after scroll
+        this.updateMethod = this.getUpdateMethod();
+
+        mediator.on('window:scroll', _.throttle(function () {
+            this[this.updateMethod]();
```

Comment:
there is no scroll event listener on Profile pages, because nav should be sticky from the beginning